content/en/docs/Architecture/meta.md

```diff
@@ -21,9 +21,14 @@ Every component is extensible and modular such as it can be customized and repla
 
 In order to convert a Linux Distribution to Kairos, the distribution must meet the following requirements:
 
-- Either use `systemd` or another init system
-- Use `grub` as bootloader
-- If the system supposedly is meant to be used with EFI, the kernel needs to have enabled the `CONFIG_EFI_STUB` option ( see: https://docs.kernel.org/admin-guide/efi-stub.html)
+- Trusted Boot Images
+  - Use a recent enough version of `systemd` as init system
```

Comment:
can we exactly specify the version here?

for Kairos 3.0.0 the minimum is systemd 252
Then for 3.1.x or 3.2.x we raised it to minimum of 256 due to bugs in the tpm stuff

Reply:
Very good point, I'll add it. I will only mention 256 then because now we have versioned documentation

Reply:
Very good point, I'll add it. I will only mention 256 then because now we have versioned documentation